tabulator/stream.py

```diff
@@ -84,9 +85,10 @@ class Stream(object):
             to False.
 
         skip_rows (List[Union[int, str]], optional):
-            List of row numbers and
-            strings to skip. If a string, it'll skip rows that begin with it
-            (e.g. '#' and '//').
+            List of row numbers, strings and regex patterns to skip.
+            If a string, it'll skip rows that begin with it e.g. '#' and '//'.
```

Comment:
'rows that begin with' is a bit ambiguous - perhaps 'rows that their first cells begin with the string or match the regex'